packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -104,14 +108,16 @@ _The key elements of the component's public API surface:_
     -   `dieLabelsSuffix` - represent a string that can be added as a label in the end of the each data information in the wafer map dies value.
     -   `colorScaleMode` - represent an Enum value that determent if the colorScale is represent a continues gradient values (linear), or is set categorically (ordinal).
     -   `highlightedValues` - represent a list of strings of dies values that will be highlighted in the wafer map view
-    -   disabled - it's represented by a boolean value and refers to the state of the `nimble-wafer-map` component. If true, the component should be rendered dimmed out and no user interaction should be allowed.
+    -   `disabled` - it's represented by a boolean value and refers to the state of the `nimble-wafer-map` component. If true, the component should be rendered dimmed out and no user interaction should be allowed.
+    -   `validity` - readonly object of boolean values that represents the validity states that the wafer map's configuration can be in. The object's type is WaferMapValidity, and it contains the following boolean properties:
+        -   `invalidDieGridDimensions` : true when only one or two of the `dieGridOriginX`, `dieGridOriginY`, `dieGridRows` or `dieGridCols` is `undefined`.
```

Comment:
I'm not sure this description is accurate for when this validity state would be `true`. Why would this be `true` based on any subset of these attributes being undefined, when there seems to be a well-defined meaning to what undefined results in for each property (which doesn't seem like it would be in conflict with its associated property that has a non-undefined value)?

Wouldn't we want this to be `true` only if `dieGridOriginX` was more than `dieGridCols` (and similar for the other properties)? I'm not sure we would say it's `true` even in the scenario that the provided values provided a coordinate space where nothing in the `dies` mapped to (but perhaps we could/should?).

If I'm not understanding something please let me know.

Reply:
I mentioned that when any subset of these attributes is undefined the validity flag will be raised because in that case the provided bounding box is obviously incomplete, so we are not going to use it to display the wafer. @rajsite mentioned that in this case is customary to not display anything and to raise the error flag for debugging

The `dieGridOriginX` value being greater than `dieGridCols` is not a concern because the columns are counted starting from the originX position, so if that position is 7 and we have 3 columns, the last column will be in the 9 position.

There still is the case you mentioned in which the provided bounding box does not cover any actual data although it will appear valid. Do you think this is something we should check internally, or should showing just the empty display align better with the nimble policy? If we are going to check if the input data aligns with the bounding box it may cost us some performance.